### ISSUE TITLE
Weird `Params` for function completion

### ISSUE THREAD

**Issue by Carreau** [davidhalter/jedi#924]
```
In [11]: Script('bi').completions()[0].params
Out[11]: [<_Param param 2796202>]
````

function is `bin` which signature is `bin(number, /)` the `/` being end of positional-only arguments that can be done only using C. the `2796202` is still a bit weird :-)

**Comment by davidhalter** [comment]
We're currently not using `inspect.signature(bin)`, but a weird way of `__doc__` parsing. We should just use the signature call whenever it's available.

**Comment by davidhalter** [comment]
I fixed this in d1c85191a0931bfa5dd0773d6d2f754199ee9699 by using `inspect.signature`.
